<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui3DMainWindowClass</class>
 <widget class="QMainWindow" name="Gui3DMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterVisualizers">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter2D3D">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab3D">
         <attribute name="title">
          <string>3D View</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab2D">
         <attribute name="title">
          <string>Image View</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QScrollArea" name="imgScrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="imgScrollAreaContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="imgPlaceholder">
                <property name="lineWidth">
                 <number>0</number>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QFrame" name="sd">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="imgPlaceholder2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="controlsPlaceholder">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionChangeView"/>
    <addaction name="actionShowHideControlPanel"/>
    <addaction name="separator"/>
    <addaction name="actionWhiteBackground"/>
    <addaction name="actionBlackBackground"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menuGrab">
    <property name="title">
     <string>&amp;Grab</string>
    </property>
    <addaction name="actionSetOutputDirectory"/>
    <addaction name="actionSetFilePattern"/>
    <addaction name="separator"/>
    <addaction name="actionGrab"/>
    <addaction name="actionShot"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuGrab"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWhiteBackground">
   <property name="text">
    <string>Set White Background</string>
   </property>
  </action>
  <action name="actionBlackBackground">
   <property name="text">
    <string>Set Black Background</string>
   </property>
  </action>
  <action name="actionSetOutputDirectory">
   <property name="text">
    <string>Set Output Directory</string>
   </property>
  </action>
  <action name="actionSetFilePattern">
   <property name="text">
    <string>Set File Pattern</string>
   </property>
  </action>
  <action name="actionGrab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionShot">
   <property name="text">
    <string>Shot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShowHideControlPanel">
   <property name="text">
    <string>Show/Hide Control Panel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionChangeView">
   <property name="text">
    <string>Change View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
